build.gradle

```diff
@@ -195,20 +204,39 @@ project(':kcbq-connector') {
         compile (
                 project(':kcbq-api'),
 
-                "com.google.cloud:google-cloud:$googleCloudVersion",
+                "com.google.cloud:google-cloud-bigquery:$googleCloudVersion",
+                "com.google.cloud:google-cloud-storage:$googleCloudVersion",
                 "com.google.auth:google-auth-library-oauth2-http:$googleAuthVersion",
-                "com.google.code.gson:gson:$googleCloudVersion",
+                "com.google.code.gson:gson:$googleCloudGsonVersion",
                 "io.debezium:debezium-core:$debeziumVersion",
+
+                "io.confluent:kafka-connect-avro-converter:$ioConfluentVersion",
+                "io.confluent:kafka-schema-registry-client:$ioConfluentVersion",
+                "org.apache.avro:avro:$avroVersion",
                 "org.apache.kafka:connect-api:$kafkaVersion",
                 "org.apache.kafka:kafka-clients:$kafkaVersion",
                 "org.slf4j:slf4j-api:$slf4jVersion",
                 "org.slf4j:slf4j-simple:$slf4jVersion",
+                "com.jayway.jsonpath:json-path:$jsonPathVersion",
+                "ch.qos.logback:logback-core:$logbackVersion",
+                "log4j:log4j:$log4jVersion"
```

Comment:
I don't think we should add logback or log4j as a dependency.  We want to let the user pick their logging implementation.